subworkflows/plink.cwl

```diff
@@ -0,0 +1,110 @@
+cwlVersion: v1.2
+class: Workflow
+id: plink
+doc: |
+  Plink subworkflow. Loads a VCF then runs genome and sexchecks on it.
```

Comment:
sexchecks vs sex checks?

Reply:
sexcheck is the name of the flag. I'll remove this since we're no longer running it anyway.

Reply:
Ok. Removed that bit.